server/src/test/java/org/opensearch/common/settings/WriteableSettingTests.java

```diff
@@ -462,6 +463,7 @@ public void testVersionSetting() throws IOException {
         }
     }
 
+    @Ignore
```

Comment:
You can fix this test in the following way: Add a setter marked as package-private (no access flags) in the s
Settings class to modify the default. By that only a class from the same package can modify the default value, not code anywjere outside.

WARNING: If the fields to modify are final, don't test this at all, kill this test!